app/code/community/Metrilo/Analytics/controllers/Adminhtml/AjaxController.php

```diff
@@ -92,7 +92,6 @@ public function indexAction()
                     $result['success'] = false;
                     break;
             }
-            $result['success'] = true;
```

Comment:
Why this gone, hayaaa?

Reply:
Because this is a mistake that sets all of the returned api connection codes to true, therefore failed api request will be marked as true, witch is wrong.